rust/processor/src/processors/default_processor.rs

```diff
@@ -76,76 +82,84 @@ async fn insert_to_db(
         "Inserting to db",
     );
 
-    let txns_res = execute_in_chunks(
-        conn.clone(),
-        insert_transactions_query,
-        txns,
-        get_config_table_chunk_size::<TransactionModel>("transactions", per_table_chunk_sizes),
-    );
-    let bmt_res = execute_in_chunks(
+    execute_in_chunks(
         conn.clone(),
         insert_block_metadata_transactions_query,
         block_metadata_transactions,
         get_config_table_chunk_size::<BlockMetadataTransactionModel>(
             "block_metadata_transactions",
             per_table_chunk_sizes,
         ),
-    );
-    let wst_res = execute_in_chunks(
-        conn.clone(),
-        insert_write_set_changes_query,
-        wscs,
-        get_config_table_chunk_size::<WriteSetChangeModel>(
-            "write_set_changes",
-            per_table_chunk_sizes,
-        ),
-    );
-    let mm_res = execute_in_chunks(
+    )
+    .await?;
+
+    execute_in_chunks(
         conn.clone(),
         insert_move_modules_query,
         move_modules,
         get_config_table_chunk_size::<MoveModule>("move_modules", per_table_chunk_sizes),
-    );
-
-    let mr_res = execute_in_chunks(
-        conn.clone(),
-        insert_move_resources_query,
-        move_resources,
-        get_config_table_chunk_size::<MoveResource>("move_resources", per_table_chunk_sizes),
-    );
+    )
+    .await?;
 
-    let ti_res = execute_in_chunks(
+    execute_in_chunks(
         conn.clone(),
         insert_table_items_query,
         table_items,
         get_config_table_chunk_size::<TableItem>("table_items", per_table_chunk_sizes),
-    );
+    )
+    .await?;
 
-    let cti_res = execute_in_chunks(
+    execute_in_chunks(
         conn.clone(),
         insert_current_table_items_query,
         current_table_items,
         get_config_table_chunk_size::<CurrentTableItem>(
             "current_table_items",
             per_table_chunk_sizes,
         ),
-    );
+    )
+    .await?;
 
-    let tm_res = execute_in_chunks(
+    execute_in_chunks(
         conn.clone(),
         insert_table_metadata_query,
         table_metadata,
         get_config_table_chunk_size::<TableMetadata>("table_metadatas", per_table_chunk_sizes),
-    );
+    )
+    .await?;
 
-    let (txns_res, bmt_res, wst_res, mm_res, mr_res, ti_res, cti_res, tm_res) =
-        join!(txns_res, bmt_res, wst_res, mm_res, mr_res, ti_res, cti_res, tm_res);
+    if !deprecated_tables.contains(&"transactions".to_string()) {
```

Comment:
Adding the deprecation here means that we're still processing the models though.

I would think that the better way would be having a function in the structs that says fn should_process that gets called in the processor code.

Reply:
yeah that's what I meant in the standup we can avoid processing the models instead of just not write to db.

Reply:
I suggested to merge this one with struct parsing but not DB writing, and then follow up with one that disables them further upstream after